egui_glow/src/epi_backend.rs

```diff
@@ -74,9 +97,9 @@ fn integration_info(
 }
 
 // ----------------------------------------------------------------------------
-
+#[cfg(not(feature = "painter_only"))]
 pub use epi::NativeOptions;
-
+#[cfg(not(feature = "painter_only"))]
```

Comment:
Maybe put `#[cfg(not(feature = "painter_only"))]` over `mod epi_backend;` instead of adding it to (almost) everything in this file?

Reply:
add `winit` feature instead of not(feature="painter_only").